scripts/release/rpm/test_mariner_in_docker.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# This script should be run in a centos7 docker.
+set -exv
+
+export USERNAME=azureuser
+
+tdnf --nogpgcheck install /mnt/rpm/$RPM_NAME -y
+
+tdnf install git gcc python3-devel python3-pip findutils -y
```

Comment:
There are two differences with `test_rpm_in_docker.sh`
1. Mariner does not install `pip` when install `python3.9-devel`, it also does not have `python3.9-pip` package.
2. `dnf` becomes `tdnf`.

I don't want to add too many env variables, so I create a new script for Mariner.